### ISSUE TITLE
Unable to run Stryker on react type script project--Getting error

### ISSUE THREAD

**Issue by JoyalJoyMadeckal** [stryker-mutator/stryker#1691]
I have configured Stryker for my project in React Type script. But unable to generate the results. Its throwing me error on running (Emit Error.)
This is my project where i am trying to run. Can anyone suggest me a solution for this.
[joyal.zip](https://github.com/stryker-mutator/stryker/files/3556305/joyal.zip)

**Comment by nicojs** [comment]
Hi @JoyalJoyMadeckal 👋 I've managed to get it working ❤

First of all, there is an issue in more recent versions of `react-scripts`. It doesn't support directories starting with a `.`. For example `.stryker-tmp`, the temp directory Stryker creates to run its sandboxes 🤦‍♂️. See https://github.com/facebook/create-react-app/issues/7660

Luckily, we've just merged configurable temp directory: #1644. It is released with stryker `2.1.0`. 

Update your stryker dependencies:

```
"@stryker-mutator/core": "^2.1.0",
"@stryker-mutator/jest-runner": "^2.1.0",
"@stryker-mutator/typescript": "^2.1.0",
```

And add this to your `stryker.conf.js`

```
 tempDirName: 'stryker-tmp',
```

There was one more issue. It has to do with your css files. They are generated and ignored with `.gitignore`. Therefore, Stryker doesn't know to include them. So either remove `*.css` from `.gitignore` (not ideal) or override the `files` property in stryker.conf.js:

```
    files: [
      'src/**',
      'public/**'
    ],
```

Result here:
[joyal.zip](https://github.com/stryker-mutator/stryker/files/3587561/joyal.zip)

Have a lot of Joy mutation testing!
